release-notes/release-notes57.md

```diff
@@ -187,6 +188,27 @@ The {{site.productname}} 5.7 release includes an accompanying release of the **P
 
 For information on using premium skins and icon packs, see: [Premium Skins and Icon Packs]({{site.baseurl}}/enterprise/premium-skins-and-icon-packs/).
 
+## Accompanying Premium self-hosted server-side component changes
+
+The {{site.productname}} 5.7 release includes accompanying changes affecting the {{site.productname}} **self-hosted** services for the following plugins:
+
+- The Enhanced Media Embed plugin (`mediaembed`)
+- The Image Tools plugin (`imagetools`)
+- The Link Checker plugin (`linkchecker`)
+- The Spell Checker Pro plugin (`tinymcespellchecker`)
+
+The Java server-side components (`ephox-spelling.war`, `ephox-hyperlinking.war`, and `ephox-image-proxy.war`) have been updated to **version 2.102**.
+
+This version upgraded and replaced internal dependencies to provide upstream bug fixes and smaller `.war` archives.
```

Comment:
smaller war archives wasn't really an aim, but the aims were complicated. Probably just "upgrading dependencies that were really out of date". I guess the war file size was sort of relevant in the moving to cats from scalaz, but the move wasn't really prompted by code size.

However, it's good enough. for release notes. Essentially "upgraded dependencies" ... the main driver being to modernise the codebase.

Reply:
Yeah, I figured it wasn't the primary aim, however it was a side-effect that I thought was worth calling out :shrug: